projects/m2/023-scrabble-score/js/index.js

```diff
@@ -0,0 +1,30 @@
+const points = {
+    1: ["A", "E", "I", "L", "N", "O", "R", "S", "T", "U"],
+    2: ["D", "G"],
+    3: ["B", "C", "M", "P"],
+    4: ["F", "H", "V", "W", "Y"],
+    5: "K",
+    8: ["J", "X"],
+    10: ["Q", "Z"]
+}
+
+const string = "hello"
+
+
+function scrabblePoint(p, s) {
+    let score = 0
+    for (i = 0; i < s.length; i++) {
+        const upper = s[i].toUpperCase()
+        for (k in p) {
+            if(p[k].includes(upper)){
+                score += Number(k);
+            }
```

Comment:
If you want, you can try to solve this exercise using the `entries` method:
https://github.com/tomorrowdevs-projects/programming-basics/pull/3299#discussion_r1257499815